src/compatTest/groovy/io/github/cosmicsilence/scalafix/ScalafixPluginFunctionalTest.groovy

```diff
@@ -53,6 +67,18 @@ class ScalafixPluginFunctionalTest extends Specification {
         buildResult.output.contains(':scalafixTest SKIPPED')
     }
 
+    def 'scalafixMain task should log a message warning the user scalafix.autoConfigureSemanticdb is deprecated when \
+when scalafix.autoConfigureSemanticdb is disabled'() {
```

Comment:
could we make it shorter? E.g. `scalafixMain task should log a deprecation warning the user autoConfigureSemanticdb is used`

Reply:
actually, could this be part of the test above? It's a bit expensive to run these integration tests + this test will be deleted eventually

Reply:
You're right on the 2nd comment. I will make it shorter, but the right test is actually `scalafixMain task should log a deprecation warning when autoConfigureSemanticdb is set to false` if that field is null or true we are just doing whatever `semanticdb.autoconfigure` commands :)